utils/config/daos_server.yml

```diff
@@ -204,7 +203,7 @@
 #
 #  targets: 20
```

Comment:
While you are there, could you please change number of targets in this file to 16 instead of 20?